### PR TITLE
[#31] Remove usage of client_id now since 1 app

### PR DESCRIPTION
**Pull request by PLeVasseur**
* Update UMessage Request -> vsomeip msg to store source UUri.
* Update vsomeip msg Response -> UMessage to use cached Request's source UUri.
* Update vsomeip msg Request -> UMessage to use the configured ue_id and ue_version_major passed into construction of UPTransportVsomeip.

Implements [#30]

### CODE REVIEW

File: up-transport-vsomeip/src/message_conversions.rs [hunk 1]
```diff
@@ -540,11 +513,13 @@ impl VsomeipMessageToUMessage {
             UP_CLIENT_VSOMEIP_FN_TAG_CONVERT_VSOMEIP_MSG_TO_UMSG,
             request_id
         );
-        let req_id = rpc_correlation_registry.remove_ue_request_correlation(request_id)?;
+        let (req_id, sink) = rpc_correlation_registry.remove_ue_request_correlation(request_id)?;
```
Comment (dmacattack) [inline up-transport-vsomeip/src/message_conversions.rs:516]: Sink is only used for traces?

Comment (PLeVasseur) [reply up-transport-vsomeip/src/message_conversions.rs:516]: It's used a few lines below in order to build response `UMessage` using the `UMessageBuilder`.

File: up-transport-vsomeip/src/lib.rs [hunk 2]
```diff
@@ -197,6 +197,17 @@ impl UPTransportVsomeip {
...
+        if check_ue_version_major.is_err() {
+            return Err(UStatus::fail_with_code(
+                UCode::INVALID_ARGUMENT,
+                format!(
+                    "uri's ue_version_major doesn't fit allotted 8 bits: uri.ue_version_major: {}",
+                    uri.ue_version_major
+                ),
+            ));
+        }
+
```
Comment (sophokles73) [inline up-transport-vsomeip/src/lib.rs:210]: FMPOV this check should be in up-rust. I guess that is what https://github.com/eclipse-uprotocol/up-rust/issues/209 is about, right?

**Review by sophokles73 — APPROVED** [review]
Nothing strikes me as completely off, but the usual disclaimer applies ;-)

File: up-transport-vsomeip/src/message_conversions.rs (hunk 1 above)
Comment (dmacattack) [reply up-transport-vsomeip/src/message_conversions.rs:516]: touche' my bad for code review on a phone

**Review by dmacattack — APPROVED** [review]
Looks good.
FWIW I find tuple returns get abused in languages that have them ( * cough * python ) :smile: 

I'm not saying your code applies to this anecdote. Just a warning I have observed it _can_ lead to a lazy structure, and harder to read/maintain code.

**Pull request merged**